responses/registries.py

```diff
@@ -73,11 +73,31 @@ def replace(self, response: "BaseResponse") -> "BaseResponse":
         self.registered[index] = response
         return response
 
+    def _dump(self, destination):
+        import yaml
+
+        data = {"responses": []}
+        for rsp in self.registered:
+            data["responses"].append(
+                {
+                    "response": {
+                        "method": rsp.method,
+                        "url": rsp.url,
+                        "body": rsp.body,
+                        "status": rsp.status,
+                        "headers": rsp.headers,
+                        "content_type": rsp.content_type,
+                        "auto_calculate_content_length": rsp.auto_calculate_content_length,
+                    }
+                }
+            )
+        yaml.dump(data, destination)
```

Comment:
I'm not sure yaml is the right format to use for this given how ambiguous YAML syntax is and how awkward YAML is at defining multi-line strings. TOML might be a less troublesome default, or we leave serialization undefined and users can supply their own 'save/load' implementation.

Reply:
I thought about JSON, but with JSON there is a limitation of comments. I could expect that some users may want to add some notes to the recorded files

Reply:
my usage of toml is limited to `pyproject.toml` only. Do you think this will be more powerful format?

I see some good signs for it, since in py3.10 CPy has built-in toml parser, if I remember correctly